frontend/src/types/QuestionTypes.ts

```diff
@@ -13,6 +7,29 @@ export enum QuestionElementType {
   MULTI_SELECT = "Multi-select",
 }
 
+export type ResponseElementType = Extract<
+  QuestionElementType,
+  | QuestionElementType.MULTIPLE_CHOICE
+  | QuestionElementType.SHORT_ANSWER
+  | QuestionElementType.MULTI_SELECT
+>;
+
+// eslint-disable-next-line @typescript-eslint/no-redeclare
+export const ResponseElementType = {
```

Comment:
is there a reason we aren't using a `ResponseElementEnum`? might be cleaner to use like `ResponseElementEnum. MULTIPLE_CHOICE ` for example in `QuestionTag.tsx` since type is restricted to `ResponseElementType`

Reply:
i.e why don't we use a separate unrelated enum entirely?

here, i've defined `ResponseElementType` as a subset of `QuestionElementType`. i think this is beneficial in cases like `generateQuestionCardTags()` where it can take elements from the question editor (`QuestionElementType`) and map it to a specific `QuestionTag` (of `ResponseElementType).

let me know if that makes sense or if i've misunderstood your comment!

Reply:
ah ok - i guess just in general i was thinking if we should have two enums for "Prompt" types (Question, Text, Image) and "Response" types and then have a union of the two 

but if there's no use for singling out the prompt types then we can keep as is